src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/ExpenditureSettings.tsx

```diff
@@ -0,0 +1,247 @@
+  FormSection,
+} from '~core/Fields';
+import { useLoggedInUser, useColonyFromNameQuery } from '~data/index';
+import Numeral from '~core/Numeral';
+
+import styles from './ExpenditureSettings.css';
+import UserAvatar from '~core/UserAvatar';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+
+import { tokens as tokensData } from './consts';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import UserMention from '~core/UserMention';
+import DomainDropdown from '~core/DomainDropdown';
+import ColorTag, { Color } from '~core/ColorTag';
+import { SpinnerLoader } from '~core/Preloaders';
+
+const MSG = defineMessages({
+  typeLabel: {
+    id: 'dashboard.Expenditures.ExpenditureSettings.defaultExpenditureLabel',
+    defaultMessage: 'Expenditure type',
+  },
+  teamLabel: {
+    id: 'dashboard.Expenditures.ExpenditureSettings.defaultTeamLabel',
+    defaultMessage: 'Team',
+  },
+  balanceLabel: {
+    id: 'dashboard.Expenditures.ExpenditureSettings.defaultBalanceLabel',
+    defaultMessage: 'Balance',
+  },
+  ownerLabel: {
+    id: 'dashboard.Expenditures.ExpenditureSettings.defaultOwnerLabel',
+    defaultMessage: 'Owner',
+  },
+  optionAdvanced: {
+    id: 'dashboard.Expenditures.ExpenditureSettings.defaultAdvancedOption',
+    defaultMessage: 'Advanced payment',
+  },
+});
+
+interface Props {
+  colonyName: string;
+}
+
+const ExpenditureSettings = () => {
+  const { walletAddress, username } = useLoggedInUser();
+
+  const { colonyName } = useParams<{
+    colonyName: string;
+  }>();
+
+  const { data: colonyData, loading } = useColonyFromNameQuery({
+    variables: { address: '', name: colonyName },
+  });
+
+  const getDomainColor = useCallback<(domainId: string | undefined) => Color>(
+    (domainId) => {
+      const rootDomainColor: Color = Color.LightPink;
+      const defaultColor: Color = Color.Yellow;
+      if (domainId === String(ROOT_DOMAIN_ID)) {
+        return rootDomainColor;
+      }
+      if (!colonyData?.processedColony || !domainId) {
+        return defaultColor;
+      }
+      const domain = colonyData?.processedColony?.domains.find(
+        ({ ethDomainId }) => Number(domainId) === ethDomainId,
+      );
+      return domain ? domain.color : defaultColor;
+    },
+    [colonyData],
+  );
+
+  const renderActiveOption = useCallback<
+    (option: SelectOption | undefined, label: string) => ReactNode
+  >(
+    (option, label) => {
+      const value = option ? option.value : undefined;
+      const color = getDomainColor(value);
+      return (
+        <div className={styles.activeItem}>
+          <ColorTag color={color} />{' '}
+          <div className={styles.activeItemLabel}>{label}</div>
+        </div>
+      );
+    },
+    [getDomainColor],
+  );
+
+  const filterDomains = useCallback((optionDomain) => {
+    const optionDomainId = parseInt(optionDomain.value, 10);
+    if (optionDomainId === 0) {
+      return false;
+    }
```

Comment:
We have a constant for this `COLONY_TOTAL_BALANCE_DOMAIN_ID` _(although it has a bit of an awkward name)_